Library/ViewModels/CommentsViewModel.swift

```diff
@@ -39,6 +42,12 @@ public protocol CommentsViewModelOutputs {
 
   /// Emits a list of `Comment`s and the `Project` to load into the data source.
   var loadCommentsAndProjectIntoDataSource: Signal<([Comment], Project), Never> { get }
+
+  /// Emits when a comment was successfully posted.
+  var postCommentSuccessful: Signal<Comment, Never> { get }
```

Comment:
Combine this and the error output to be `var showBannerWithMessageAndType: Signal<(String, MessageBannerType), Never>`.

Reply:
I guess I can leave this to the ticket for handling showing success or failure feedback to the user.

Reply:
Be sure to mark it as `TODO` in that case and comment on the relevant ticket what needs to be done, please. I think these banners are also only shown when retrying after an error?

Reply:
Based on the design, I think banners are only shown when flagging a comment. The failed, posting and posted state are part of the comment cell design.